tests/e2e/config/bootstrap.js

```diff
@@ -161,7 +166,7 @@ beforeAll( async() => {
 	capturePageEventsForTearDown();
 	enablePageDialogAccept();
 	observeConsoleLogging();
-	if ( process.env.DEBUG_REST ) {
+	if ( +process.env.DEBUG_REST ) {
```

Comment:
IMO a strict comparison here is easier to grasp than using `+`

Reply:
```suggestion
	if ( "1" === process.env.DEBUG_REST ) {
```